apps/kitchen-sink/src/ensemble/screens/home.yaml

```diff
@@ -11,6 +11,9 @@ View:
         - Text:
             styles:
               names: heading-1
+            htmlAttributes:
+              Value: hello
+              type: kio
```

Comment:
Can you add a new section in Custom Widgets page for this? Title: Adding HTML Attributes